codingTracker.jakubmikolajewski/CodingTrackerLibrary/CodingController.cs

```diff
@@ -0,0 +1,279 @@
+﻿using Dapper;
+
+namespace CodingTrackerLibrary
```

Comment:
Remember as per the [code-conventions](https://thecsharpacademy.com/article/58/code-conventions): 
From C# 10 we are able to ditch the namespace block and save one indentation level, which optimizes the use of space.

![code-convention-6](https://github.com/user-attachments/assets/f813d72d-94f4-41d6-aec8-f37371eab50a)